tests/test_tools.py

```diff
@@ -2174,5 +2175,69 @@ def test_get_tags_invalid():
         get_tags(coll)
         assert e_info == 'ERROR: valid tags are comma or space separated strings of tag names'
 
+@pytest.mark.parametrize(
+    "inputs, expected", [
+        # everything is defined, return 1
```

Comment:
I wonder if we should change what is returned.  exit(0) usually means the function executed successfully.   It makes confusion here.  Maybe to get the intent clear we could call the function `repo_information_complete()` and have it return True or False, then when it is used in the code it will look like 
```
  if repo_information_complete():
     make my repo code
```